client/platform/desktop/frontend/components/Jobs.vue

```diff
@@ -101,6 +122,15 @@ export default defineComponent({
                           {{ job.job.datasetIds.join(', ') }}
                         </td>
                       </tr>
+                      <tr>
+                        <td>work dir</td>
+                        <td
+                          class="selectable"
+                          @click="openPath(job.job)"
+                        >
+                          show in file manager
```

Comment:
Is this intentionally styled to be somewhat hidden?  It doesn't look like a clickable area like the `Console Log` it's regular text that has a cover state.

Reply:
Better?

![Screenshot from 2021-03-08 17-40-23](https://user-images.githubusercontent.com/4214172/110391393-628d9500-8035-11eb-97a3-a21fd77fb93c.png)